docs/synchronizations-faq.md

```diff
@@ -1,27 +1,32 @@
 # Synchronizations FAQ
+##
 
 In Private Packagist synchronizations is how you keep packages, teams, their members, and access permissions in sync with your version control system like GitHub, GitLab or Bitbucket.
 
-You can find below answers for frequently-asked questions concerning Synchronizations.
+You can find below answers for frequently-asked questions concerning synchronizations.
 
 ## How to migrate my synchronization from GitHub to Bitbucket?
 
 1. Add the new synchronization from Bitbucket.
 2. Delete the packages that were created from the GitHub synchronization. You can filter by the GitHub synchronization on the package list page and then you can delete the packages from that list.
-3. Make sure the new synchronization from Bitbucket successfully run. Then you can safely delete the old synchronization from GitHub.
+3. Make sure the new synchronization from Bitbucket successfully run. In case the old synchronization was marked as primary, you need to promote the new synchronization as primary otherwise you won't be able to delete it. Then you can safely delete the old synchronization from GitHub.
```

Comment:
I wonder whether we need to explain what switching primary synchronization implies e.g. new owner team and in this particular case that users will have to connect their personal account with the new service to make sure they do not lose access?

Reply:
yes I think that's worth mentioning